frontend/src/features/public-form/components/FormPaymentPage/components/PaymentFormBannerLogo.tsx

```diff
@@ -0,0 +1,32 @@
+import { FormLogoState } from '~shared/types'
+
+import { useEnv } from '~features/env/queries'
+import { usePublicFormContext } from '~features/public-form/PublicFormContext'
+
+import { FormBannerLogo, useFormBannerLogo } from '../../FormLogo'
+
+export const PaymentFormBannerLogo = (): JSX.Element => {
+  const { form, spcpSession, handleLogout, isLoading } = usePublicFormContext()
+
+  const { data: { logoBucketUrl } = {} } = useEnv(
+    form?.startPage.logo.state === FormLogoState.Custom,
+  )
+  const hasSubmitted = true //true because payment banner logo (i.e. payment page) will show after a submission is done
```

Comment:
I would recommend comment to explain why this is always true.

On second thoughts, I think removing this line `hasSubmitted = true`, and set `showDefaultLogoIfNoLogo: true` would be sufficient, as we do want to show some logo.

Comments can be hard to get right 😄